<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_MainWindow</class>
 <widget class="QMainWindow" name="w_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Przychodnia - Serwer</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionServerStart"/>
   <addaction name="actionServerStop"/>
   <addaction name="actionServerConfigure"/>
   <addaction name="separator"/>
   <addaction name="actionDataBaseConfigure"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSerwer">
    <property name="title">
     <string>Serwer</string>
    </property>
    <addaction name="actionServerStart"/>
    <addaction name="actionServerStop"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuUstawienia">
    <property name="title">
     <string>Ustawienia</string>
    </property>
    <addaction name="actionWindowConfigure"/>
    <addaction name="actionServerConfigure"/>
    <addaction name="actionDataBaseConfigure"/>
   </widget>
   <addaction name="menuSerwer"/>
   <addaction name="menuUstawienia"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionServerStart">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/actions/icons/if_server_run_103341.png</normaloff>:/actions/icons/if_server_run_103341.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionServerStop">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/actions/icons/if_server_stop_103344.png</normaloff>:/actions/icons/if_server_stop_103344.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionServerConfigure">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/actions/icons/if_server_settings_103343.png</normaloff>:/actions/icons/if_server_settings_103343.png</iconset>
   </property>
   <property name="text">
    <string>Serwer</string>
   </property>
   <property name="toolTip">
    <string>Ustawienia serwera</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Zako≈Ñcz</string>
   </property>
  </action>
  <action name="actionWindowConfigure">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/actions/icons/if_browser_window_settings_103427.png</normaloff>:/actions/icons/if_browser_window_settings_103427.png</iconset>
   </property>
   <property name="text">
    <string>Okno</string>
   </property>
   <property name="toolTip">
    <string>Ustawienia okna</string>
   </property>
  </action>
  <action name="actionDataBaseConfigure">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/actions/icons/if_032_95930.png</normaloff>:/actions/icons/if_032_95930.png</iconset>
   </property>
   <property name="text">
    <string>Baza danych</string>
   </property>
   <property name="toolTip">
    <string>Ustawienia bazy danych</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
